### PR TITLE
UI: Remove docs tab

### PR DESCRIPTION
**Pull request by tmeasday**
## What I did

- No longer add a docs tab from the docs addon
- Always select docs `viewMode` when selecting a docs entry
- Replace `story` with `entry` in some places (should we do more of this? Should we make it `entryId` (rather than `storyId`) also?)
- Remove concept of `docsOnly` from the UI, instead e.g. hide panel 

## How to test

Run `react-ts`.

### CODE REVIEW

File: lib/core-server/src/__snapshots__/cra-ts-essentials_manager-prod-posix [hunk 1]
```diff
@@ -1,3 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`cra-ts-essentials manager dev 1`] = `null`;
```
Comment (shilman) [inline lib/core-server/src/__snapshots__/cra-ts-essentials_manager-prod-posix:3]: i think this is a race condition

**Review by shilman — APPROVED** [review]
🚀 ❤️

File: lib/core-server/src/__snapshots__/cra-ts-essentials_manager-prod-posix (hunk 1 above)
Comment (tmeasday) [reply lib/core-server/src/__snapshots__/cra-ts-essentials_manager-prod-posix:3]: Oh, right, will drop it

**Review by shilman — COMMENTED** [review]
@tmeasday Please fix CI before merging!

**Pull request merged**